service/debugger/debugger.go

```diff
@@ -620,7 +620,17 @@ func (d *Debugger) state(retLoadCfg *proc.LoadConfig) (*api.DebuggerState, error
 	return state, nil
 }
 
-// CreateBreakpoint creates a breakpoint.
+// CreateBreakpoint creates a breakpoint using information from the provided `requestedBp`.
+// This function accepts several different ways of specifying where and how to create the
+// breakpoint that has been requested.
+//
+// 1. You may provide a file:line combination and the debugger will take care of associating that with
+// the correct address in which to place the breakpoint in the running program.
```

Comment:
It might be useful to mention behavior in error cases. For example, if you specify a blank line, it won't place a breakpoint at a statement immediately following, but return an error.

Reply:
api.Breakpoint doc is giving some hints about requests. What I want to learn is the relationship between requested breakpoints and returned breakpoints/errors (yes, the error cases @polinasok  mentioned), and whether all the fields in api.Breakpoint are relevant and can be mixed.

Reply:
I agree, I'll add in some words about the error handling and the interactions between the different fields and also some more information on the returned breakpoint.